<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>309</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QPushButton" name="inputFileButton">
                <property name="geometry">
                    <rect>
                        <x>190</x>
                        <y>170</y>
                        <width>181</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Обрати вхідний файл</string>
                </property>
            </widget>
            <widget class="QPushButton" name="outputFileButton">
                <property name="geometry">
                    <rect>
                        <x>410</x>
                        <y>170</y>
                        <width>181</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Обрати вихідний файл</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="pathInputFile">
                <property name="geometry">
                    <rect>
                        <x>100</x>
                        <y>220</y>
                        <width>261</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Шлях до файлу</string>
                </property>
                <property name="readOnly">
                    <bool>true</bool>
                </property>
            </widget>
            <widget class="QLineEdit" name="pathOutputFile">
                <property name="geometry">
                    <rect>
                        <x>410</x>
                        <y>220</y>
                        <width>261</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Шлях до файлу</string>
                </property>
                <property name="readOnly">
                    <bool>true</bool>
                </property>
            </widget>
            <widget class="QLineEdit" name="filterWord">
                <property name="geometry">
                    <rect>
                        <x>260</x>
                        <y>110</y>
                        <width>261</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Вкажіть слово для фільтру</string>
                </property>
            </widget>
            <widget class="QPushButton" name="saveFileButton">
                <property name="geometry">
                    <rect>
                        <x>350</x>
                        <y>60</y>
                        <width>93</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Зберегти</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>26</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
